misk/web/tabs/config/src/components/ConfigComponent.tsx

```diff
@@ -67,16 +73,13 @@ export default class ConfigComponent extends React.PureComponent<IConfigProps> {
     const { files, status } = this.props
     return(
       <Container>
-        <h1>App: Config</h1>
+        <h1>Config</h1>
         <p>{status}</p>
         {files && files.map(f => (
-          <div>
-            <br/>
-            <h5>{f.name}</h5>
-            <code><pre>
-            {this.formattedFile(f)}
-            </pre></code>
-          </div>))
+          <ConfigOutput>
+            <h5><strong>classpath:/{f.name}</strong></h5>
```

Comment:
off topic: we should probably put the `classpath:` bit in the name itself. (And rename `files` to `resources`)

Reply:
#447